packages/link/components/ResetLink.tsx

```diff
@@ -1,27 +1,17 @@
 import React from "react";
 import { cx } from "emotion";
-import { LinkProps } from "../types";
+import { ExpandedLinkProps } from "../types";
 import { linkReset } from "../../shared/styles/styleUtils/resets/linkReset";
+import UnstyledLink from "./UnstyledLink";
 
-const ResetLink: React.StatelessComponent<LinkProps> = ({
-  children,
+const ResetLink: React.FunctionComponent<ExpandedLinkProps> = ({
   className,
-  href,
-  openInNewTab,
-  target,
-  url,
+  children,
   ...other
 }) => (
-  <a
-    href={href || url}
-    target={!target && openInNewTab ? "_blank" : target}
-    className={cx(linkReset, className)}
-    // https://web.dev/external-anchors-use-rel-noopener/
-    rel={target === "_blank" || openInNewTab ? "noopener" : undefined}
-    {...other}
-  >
+  <UnstyledLink className={cx(linkReset, className)} {...other}>
     {children}
-  </a>
+  </UnstyledLink>
 );
 
-export default ResetLink;
+export { ResetLink as default };
```

Comment:
I haven't done default exports like this before. Are there any benefits? Or just a matter of personal preference?

Either way, I'm not opposed to it.